### DISCUSSION TITLE
github search does not show result - but i know there should be

### DISCUSSION THREAD

**Discussion by jukzi** [community/community#63353] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

https://github.com/search?q=repo%3Aeclipse-jdt%2Feclipse.jdt.core+testBug178847&type=code

does not return any Result - same in a fork of that repo:
https://github.com/search?q=repo%3Ajukzi%2Feclipse.jdt.core%20testBug178847&type=code

but there is at least the code that should be found:
[public void testBug178847() throws CoreException {](https://github.com/eclipse-jdt/eclipse.jdt.core/blob/49ba6ae9ba495e6305048301f5b2e860c044cba3/org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/JavaSearchBugsTests.java#L8743)

other similar seaches do find a result:
https://github.com/search?q=repo%3Aeclipse-jdt%2Feclipse.jdt.core%20testBug68726&type=code

**Comment by mdnurulmomen** [comment]
It seems, problem with your 'JavaSearchBugsTests.java' file as no-function of this file is searchable ! You may dive-into problem with that specific file.

**Comment by look** [comment, marked as answer]
Hi @jukzi, sorry for the inconvenience. The issue is that this file is 585 KB, which is larger than our [current limit](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations) of 350 KB. Therefore the file is not indexed.

**Reply by jukzi** [reply]
Thanks for the clarification.
Any file limit restriction is bad. Can even be a security problem: Some time ago i searched github for some malware content and now i know those results are not reliable since a user could have hidden his files by just making it large.

**Reply by sfc-gh-aalin** [reply]
Are there any plans to lift or configure the indexing limit? It's hard to have confidence in the search results when you know that results are being skipped.

**Reply by look** [reply]
We realize it is unfortunate that not all files are indexed. We may increase the limit in the future, but don't have plans to do so now. We are developing tooling so users can see what files are excluded.
